### PR TITLE
🚀[Release v2.12.4 20241130] Merge into Develop

### PR DESCRIPTION
**Pull request by kcw-grunt**
Resolved `main` conflicts

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -68,8 +68,8 @@ android {
         applicationId = 'com.loafwallet'
         minSdkVersion 29
         targetSdkVersion 34
-        versionCode 20241118
-        versionName "v2.12.2"
+        versionCode 20241130
+        versionName "v2.12.4"
```
Comment (josikie) [inline app/build.gradle:72]: Why do we add 2 in every our new app version?

**Review by josikie — APPROVED** [review]
Cool! Thank you! @kcw-grunt

File: app/build.gradle (hunk 1 above)
Comment (andhikayuana) [reply app/build.gradle:72]: Wondering, is there an odd-even release strategy?

Comment (kcw-grunt) [reply app/build.gradle:72]: I'll message directly

Comment (andhikayuana) [reply app/build.gradle:72]: sure, thanks!

**Pull request merged**